signing_service/signing_service/signing_service.py

```diff
@@ -319,6 +319,13 @@ def _parse_arguments() -> argparse.Namespace:
         type=int,
         help=f'Port on which Concent cluster is listening (default: {SIGNING_SERVICE_DEFAULT_PORT}).',
     )
+    parser.add_argument(
+        '-e',
```

Comment:
Why do you add here `-e`? Will it be used in deployment?

Reply:
No, we'll use the long version but it's a good idea to always provide short versions too - for convenience in cases where you just want to launch it manually and don't care about readability.